board_rules.md

```diff
@@ -0,0 +1,73 @@
+# Board rules 
+
+ The description of the team workflow for the board.
+
+## Description
+
+The board consists of 5 columns:
+```
+**Common backlog** - includes all tasks that should be done.
+**Sprint backlog** - includes the list of tasks for the current sprint.
+**In Progress** - the list of tasks that the team works on.
+**In Review** - the list of tasks that are under review process.
+**Done** - the list of tasks that successfully merged to main branch.
+```
+
+### Common backlog
+
+The process of the issue creating and adding to Common backlog.
+Every issue is placed in a common backlog when it is created.
+The issue should contain a clear title and description with definitions of done (**DoD**).
+Select necessary labels, if applicable.
+Don't forget to link the project Repository.
+
+### Sprint backlog
+
+Before sprint planning the issues place into Sprint backlog by PM.
+During the sprint planning process, the issues from the sprint backlog are estimated (labels with estimation are assigned), and the assignee is defined. 
+
+**Estimation** is carried out with the participation of all team members online using the **Scrum Poker** technology on the resource:
+```
+ https://planningpokeronline.com
+ ``` 
+Except estimation labels in progect also provided labels 
+```high, medium, bug, blocker, enhancement, help wanted.
+```
+All types of labels are marked with separate color.
```

Comment:
It doesn't display properly. Please check the markdown format.
Maybe we need to add a description for every label, that we used.
```suggestion
Except estimation labels in progect also provided labels: 

`high`, `medium`, `bug`, `blocker`, `enhancement`, `help wanted`.

All types of labels are marked with separate color.
```

Reply:
Thanks, the description for labels is added.